expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java

```diff
@@ -41,166 +49,210 @@
 import no.unit.nva.model.testing.PublicationGenerator;
 import no.unit.nva.model.testing.PublicationInstanceBuilder;
 import nva.commons.core.paths.UriWrapper;
+import org.junit.Before;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.MethodSource;
 
 class ExpandedResourceTest {
-    
+
     public static final String PUBLISHER_NAME_JSON_PTR =
         "/entityDescription/reference/publicationContext/publisher/name";
     public static final String SERIES_NAME_JSON_PTR =
         "/entityDescription/reference/publicationContext/series/name";
-    
+
+    private UriRetriever uriRetriever;
+
+    @BeforeEach
+    void setup() {
+        this.uriRetriever = mock(UriRetriever.class);
+        when(uriRetriever.getRawContent(any(), any())).thenReturn(Optional.empty());
+    }
+
     @Test
     void shouldReturnIndexDocumentWithValidReferenceData() throws Exception {
-        
+
         final Publication publication = randomBookWithConfirmedPublisher();
         final URI seriesUri = extractSeriesUri(publication);
         final URI publisherUri = extractPublisherUri(publication);
+        final List<URI> affiliationUris = extractAffiliationsUris(publication);
         final String publisherName = randomString();
         final String seriesName = randomString();
-        
-        final UriRetriever mockUriRetriever =
-            mockPublicationChannelPublisherResponse(seriesUri, seriesName, publisherUri, publisherName);
+
+        final UriRetriever mockUriRetriever = mock(UriRetriever.class);
+        addPublicationChannelPublisherToMockUriRetriever(mockUriRetriever, seriesUri, seriesName, publisherUri,
+                                                         publisherName);
+        addAffiliationsWithParentNodeToMockUriRetriever(mockUriRetriever, affiliationUris);
+
         final ExpandedResource indexDocument = fromPublication(mockUriRetriever, publication);
         final JsonNode framedResultNode = indexDocument.asJsonNode();
-        
+
         assertEquals(publisherUri.toString(), framedResultNode.at(PUBLISHER_ID_JSON_PTR).textValue());
         assertEquals(publisherName, framedResultNode.at(PUBLISHER_NAME_JSON_PTR).textValue());
         assertEquals(seriesUri.toString(), framedResultNode.at(SERIES_ID_JSON_PTR).textValue());
         assertEquals(seriesName, framedResultNode.at(SERIES_NAME_JSON_PTR).textValue());
+
+        var hasPart = framedResultNode.findValue("https://example.org/vocab#hasPart");
+        assertNotNull(hasPart);
```

Comment:
assertThat(hasPart, CoreMatchers.not(IsEmptyString.emptyOrNullString()));
or with static imports:
assertThat(hasPart, not(emptyOrNullString()));

This one-liner does the same trick. Just for information.